src/main.rs

```diff
@@ -52,6 +52,23 @@ fn open_dit_repo() -> Result<Repository> {
 }
 
 
+/// Get a vector of commits from values
+///
+/// This function transforms values to a vector. If no values are supplied,
+/// an empty vector will be returned.
+///
+fn values_to_hashes<'repo>(repo: &'repo Repository, values: Option<Values>) -> Result<Vec<Commit<'repo>>> {
```

Comment:
The function is all about the `Values`, but the parameter for that is an `Option` - not very clean IMO.